automation/roles/authorized-keys/tasks/main.yml

```
@@ -12,7 +11,7 @@
         user: "{{ system_user.stdout }}"
         key: "{{ item }}"
         state: present
-      loop: "{{ ssh_public_keys_list | map('replace', '\"', '') | map('replace', \"'\", \"\") | list }}"
+      loop: '{{ ssh_public_keys_list | map(''replace'', ''"'', '''') | map(''replace'', "''", "") | list }}'
```

```suggestion
      loop: "{{ ssh_public_keys_list | map('replace', '\"', '') | map('replace', \"'\", \"\") | list }}"
```
The change doesn’t bring significant improvement as it replaces escaping double quotes with escaping single quotes. Since we are standardizing on double quotes, it’s better to keep the original format.

@pat-s what do you think?

---

These were automated changes, you can change/adapt any as you like. The linter will complain, if there's an issue.